components/suggest/src/store.rs

```diff
@@ -578,8 +569,7 @@ where
         // For each collection, fetch all records
         for (collection, record_types) in record_types_by_collection {
             breadcrumb!("Ingesting collection {}", collection.name());
-            let records =
-                write_scope.write(|dao| self.settings_client.get_records(collection, dao))?;
+            let records = write_scope.write(|_| self.settings_client.get_records(collection))?;
```

Comment:
```rust
            let records = self.settings_client.get_records(collection)?;
```

No need for the write scope anymore since we don't need to write to the DB when getting records. (Sorry for not catching that before)